html/captive-portal/lib/captiveportal/PacketFence/Controller/Authenticate.pm

```diff
@@ -167,6 +167,7 @@ sub login : Local : Args(0) {
 
         # External authentication
         $c->forward('validateLogin');
+        $c->forward('testLoginRetryLimit');
```

Comment:
Would be more enforceLoginRetryLimit since that method detaches if the user should be locked